resume.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Resume</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-bar2" >
+        <ul>
+            <li> <a class="about me"href="#">About Me</a></li>
+            <li> <a class="resume"href="#">Resume</a></li>
+            <li> <a  href="">Projects</a></li>
+            <li> <a  href="">Contact</a></li>
+        </ul>
+    </header>
+  <main class="r">
+    <div class="r-header">
+        <div class="flex align-c">
+            <div class="box"></div>
+            <h2>Resume</h2>
+        </div>
+    </div>
+    <div class="r-content">
+        <div class="flex justify-b rc-header align-c">
+            <h3>Education</h3>
+            <a href="./">DOWNLOAD CV</a>
+        </div>
+        <div class="rc-content">
+            <div class="grid-rc">
+                <div class="r-left">
+                    <h2>
+                        2019 - Present
+                    </h2>
+                    <h5>
+                        JOB POSITION
+                    </h5>
+                    <h6>
+                        Company Name
+                    </h6>
+                </div>
+                <div class="r-right">
+                    <p>
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Exercitationem soluta pariatur quod inventore libero. Nam non, laborum facere aspernatur ipsam eveniet ea necessitatibus modi, ducimus error optio, sit dolore. Minus.
+                    </p>
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum consectetur commodi placeat obcaecati ut veritatis labore laboriosam facere cumque quae amet, tempora ratione magnam? Vero consequuntur maxime tenetur libero ea?</p>
+                </div>
+            </div>
+             <div class="grid-rc">
+                <div class="r-left">
```

Comment:
I like the class names here, quite descriptive, might make it just left and right, the style could be reused for the projects page